projects/quml-library/src/lib/player/player.component.html

```diff
@@ -34,11 +34,11 @@
             [maxScore]="maxScore" [time]="timeLimit" [contentName]="contentName"></quml-startpage>
         </slide>
         <slide *ngFor="let question of questions; let i= index">
-          <div *ngIf="question?.qType === 'MCQ'">
+          <div *ngIf="question?.primaryCategory === 'Practice Question Set'">
```

Comment:
both convert to lower case and compare like

question?.primaryCategory.toLowerCase() === 'practice question set'